apps/service-portal/src/components/Header/Header.tsx

```diff
@@ -113,7 +113,7 @@ export const Header = ({ position }: Props) => {
                               size="small"
                               type="span"
                               variant="utility"
-                              unfocusable
+                              aria-label={formatMessage(m.openDocuments)}
```

Comment:
You will get double foucs here 👀

Reply:
Ohno, thougt I was fixing the focus issue we talked about a few weeks ago 🙈